examples/quickstart-pytorch/client.py

```diff
@@ -97,7 +97,7 @@ def apply_transforms(batch):
 # Get node id
 parser = argparse.ArgumentParser(description="Flower")
 parser.add_argument(
-    "--node-id",
+    "node-id",
     choices=[0, 1, 2],
     type=int,
```

Comment:
```suggestion
    type=int,
    required=True,
```

Reply:
This is what i meant when i commented on your issue. With this, if someone runs the code without passing `--node-id` an error will be thrown.